PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -139,7 +145,7 @@ private void sendPayPalRequest(
         internalPayPalClient.sendRequest(activity, payPalRequest, (payPalResponse, error) -> {
             if (payPalResponse != null) {
                 String analyticsPrefix = getAnalyticsEventPrefix(payPalRequest);
-                braintreeClient.sendAnalyticsEvent(String.format("%s.app-switch.started", analyticsPrefix));
+                braintreeClient.sendAnalyticsEvent(String.format("%s.started", analyticsPrefix));
```

Comment:
Can we remove this `getAnalyticsEventPrefix()` method and just use raw strings like the other events? Looks like we're only using it once, here.

Reply:
Yep, we can simplify this and just do the switch on the line above so we can remove the `getAnalyticsEventPrefix` method entirely.